haystack/reader/farm.py

```diff
@@ -603,6 +606,30 @@ def _extract_answers_of_predictions(self, predictions: List[QAPred], top_k: Opti
 
         return answers, max_no_ans_gap
 
+    def calibrate_confidence_scores(
+            self,
+            document_store: BaseDocumentStore,
+            device: str,
```

Comment:
Do we actually need the `device` here as mandatory ar or better so say: do we need it in eval()? The reader should already be placed on GPU or CPU from the init. We might only need `device` if we want to change it :thinking:

Reply:
You are right. I added an issue #1137